src/Storages/StorageProxy.h

```diff
@@ -152,7 +152,7 @@ class StorageProxy : public IStorage
     CheckResults checkData(const ASTPtr & query, ContextPtr context) override { return getNested()->checkData(query, context); }
     void checkTableCanBeDropped([[ maybe_unused ]] ContextPtr query_context) const override { getNested()->checkTableCanBeDropped(query_context); }
     bool storesDataOnDisk() const override { return getNested()->storesDataOnDisk(); }
-    Strings getDataPaths() const override { return getNested()->getDataPaths(); }
+    Strings getDataPaths() const override { return {}; }
```

Comment:
But it will return wrong result

Reply:
As I understood `StorageProxy` keeps tables based on table functions and  with `StorageMaterializedMySQL` engine. Seems like the second one may have its own data. Can we always return empty value for `StorageTableFunctionProxy` class?

Reply:
I'm not sure, see, for example, table function `file`

Reply:
`file` function have not empty value:``